### ISSUE TITLE
[Accessibility][Labels or Instructions - Save as]: Screen reader does not associate the table labels while reading the table contents.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2026]
**User Experience:**
Screen reader users will not get proper information and may get difficulty in performing the task efficiently if table Labels are not associated with there respective content.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Navigate on homepage with tab key up to 'Save as' button and press enter on it.
3. Observe the issue on table.

**Actual Result:**
While navigating within table using ctrl+Alt+arrow keys, Screen reader does not associate the table labels i.e. Name, data modified, while reading the table contents.

**Note:**
This issue is repro on every table throughout this application.

**Expected Result:**
While navigating within table using ctrl+Alt+arrow keys, Screen reader should associate the table labels i.e. Name, data modified, while reading the table contents.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Attachment:**
[MAS3.3.2_Save as.zip](https://github.com/microsoft/BotFramework-Composer/files/4229356/MAS3.3.2_Save.as.zip)
[MAS3.3.2_Save as _tablecontents.zip](https://github.com/microsoft/BotFramework-Composer/files/4229393/MAS3.3.2_Save.as._tablecontents.zip)

**Comment by corinagum** [comment]
PR above is a partial fix. @liweitian could you link the bug you filed with the fabric team on this issue for tracking?

**Comment by liweitian** [comment]
> PR above is a partial fix. @liweitian could you link the bug you filed with the fabric team on this issue for tracking?

https://github.com/microsoft/fluentui/issues/12701. He gives a "solution" https://codepen.io/slimemold/pen/rNOxrjd?editors=0110. But we go to row first then go inside to cell. Is this behavior acceptable? You can try it. We are discussing issue and requirements.

**Comment by corinagum** [comment]
@liweitian thanks for the link. Yes, I think this is a great solution. I don't see any problems with it. :)

**Comment by ashish315** [comment]
@cwhitten As checked the fix seems good but the issue is not resolved for every table such as Recent Bot table, Notification etc

**Comment by cwhitten** [comment]
@ashish315 the bug is to Save As, not Recent Bots or Notification.

**Comment by ashish315** [comment]
@cwhitten In the issue, the note section was their according to which issue observed for each and every table.

**Comment by liweitian** [comment]
> @cwhitten In the issue, the note section was their according to which issue observed for each and every table.

Just opened a pr to solve this problem. #2740

**Comment by ashish315** [comment]
As checked issue is resolved, now the narrator narrates descriptive information about table header. So, HCL is closing the bug
